added access token expiry and refresh token expiry

sadashiv-sumo opened the pull request:
Modified access-token-expiry (1440 minutes) and refresh-token-expiry (10080 minutes) in the flag section and added them to the General Configuration.

<!--- Thanks for submitting a pull request! -->
<!--- If you haven't yet, please read the contributing guide at: -->
<!--- https://github.com/sensu/sensu-docs/blob/main/CONTRIBUTING.md -->
<!--- Provide a general summary of your changes in the Title above. -->

## Description
<!--- Describe your changes in detail -->

## Motivation and Context
<!--- Why is this change required? What problem does it solve? -->
<!--- If it fixes an open issue, please link to the issue here: "Closes #555" -->

## Review Instructions
<!--- Optional -->

sadashiv-sumo committed:
added access token expiry and refresh token expiry

Modified access-token-expiry (1440 minutes) and refresh-token-expiry (10080 minutes) in the flag section and added them to the General Configuration.

nasirhussenm reviewed (commented):
Please change the position of refresh-token so that it appears below --platform-metrics-logging-interval string

sadashiv-sumo committed:
updated sensu-docs

As per review comments updated docs

sadashiv-sumo commented on content/sensu-go/6.13/observability-pipeline/observe-schedule/backend.md:
Pushed a new commit to address feedback. Please review.

sadashiv-sumo committed:
Added the 16.13.1 release notes to the Sensu documentation and deleted the Tessen reference tab.

Added the 16.13.1 release notes to the Sensu documentation and deleted the Tessen reference tab.

jpipkin1 commented:
@jhenderson-pro, does this PR need to be approved and published?

nasirhussenm reviewed (approved):
Looks good to me

jhenderson-pro reviewed (approved):
LGTM!
